docs/docs/python-sdk/sdk_ref/infrahub_sdk/analyzer.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: analyzer
+sidebarTitle: analyzer
+---
+
+# `infrahub_sdk.analyzer`
+
+## Classes
+
+### `GraphQLQueryVariable` <sup><a href="https://github.com/opsmill/infrahub-sdk-python/blob/main/infrahub_sdk/analyzer.py#L20" target="_blank"><Icon icon="github" style="width: 14px; height: 14px;" /></a></sup>
```

Comment:
There are a lot of these links to the `main` branch along with a line number. Looking at the published docs https://5390d612.infrahub-sdk-python.pages.dev/python-sdk/sdk_ref/infrahub_sdk/analyzer#graphqlqueryvariable- I can't actually see the these links on the page. But they are available in the source. I think we'd need to get rid of them to avoid conflicts as well as having a link that points to an incorrect branch.

If possible it could make sense to have this point to the current version at build time so that we have a link to a tagged release instead of a branch where the line numbers as well as overall structure of the file might change.

Reply:
I guess it is about docusaurus configuration because the [FastMCP doc](https://gofastmcp.com/python-sdk/fastmcp-exceptions) partake of those links while they don't appear in our documentation.

Mdxify lib provides us with a CLI arg to properly point at the desired git branch (might also work with any git ref but not sure about this) in generated code links. Do you want to leverage this ?